ckan/pastertemplates/__init__.py

```diff
@@ -51,6 +51,9 @@ class CkanextTemplate(Template):
     def check_vars(self, vars, cmd):
         vars = Template.check_vars(self, vars, cmd)
 
+        reload(sys)
+        sys.setdefaultencoding('utf-8')
```

Comment:
yikes, this could have really surprising side effects. I think we should fix our bytes/unicode confusion issues where they lie instead of with a global flag like this.